test/vfd_swmr_log_writer.c

```diff
@@ -0,0 +1,2929 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * Copyright by The HDF Group.                                               *
+ * All rights reserved.                                                      *
+ *                                                                           *
+ * This file is part of HDF5.  The full HDF5 copyright notice, including     *
+ * terms governing use, modification, and redistribution, is contained in    *
+ * the COPYING file, which can be found at the root of the source code       *
+ * distribution tree, or in https://www.hdfgroup.org/licenses.               *
+ * If you do not have access to either file, you may request a copy from     *
+ * help@hdfgroup.org.                                                        *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+/* Description of this program:
```

Comment:
Does this test do anything in particular to verify the logging feature?

Also, if this test is the same as the group performance test, instead of adding this new test, maybe add an option to the group performance test to enable the logging feature?

Reply:
I like this idea as long as you and John are comfortable if I put this into the performance test. Please verify so I can merge this test to the group performance test.

Reply:
I am ok with that.  Maybe check with John also.

Reply:
Sent an email to John. Haven't received his reply. Think this should be OK to him. Go ahead to merge it.